encrypted-media/drm-keystatuses-multiple-sessions.html

```diff
@@ -39,11 +39,12 @@
         .then( function( contents ) {
 
             // Select a content item with multiple keys and initData
-            var contentitem = contents.filter( function( item ) { return item.keys.length > 1 && item.initDataType && !item.associatedInitData; } )[ 0 ];
+            var contentitem = contents.filter( function( item ) { return item.keys.length > 1 && item.initDataType && !item.associatedInitData; } )[ 0 ],
```

Comment:
It's @mwatson2 style - I didn't find any WPT style guidelines on this kind of thing.